src/components/LoginForm.vue

```diff
@@ -9,17 +9,18 @@
       {{ i18n.t("welcome") }}
     </h1>
 
-    <v-input
+    <wc-input
       class="login-form__input"
       label="Email"
       name="email"
+      id="email"
       :invalid="!!emailError"
       v-model="email"
     >
-      <template #error>
+      <wc-error v-if="emailError" slot="error">
```

Comment:
Input shows an error when it's invalid. We don't need to control this.
```suggestion
      <wc-error slot="error">
```